bucket/caesium-image-compressor.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "http://saerasoft.com/caesium",
```

Comment:
Please put items in the order of `version`, `description`, `homepage`, `license`. See other existing packages for example.